### PR TITLE
CHEMH-247: updated to the not pro font and updated decanter version

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Removed the direct reference to Source Sans Pro
- Updated it to the sans variable from Decanter
- Updated Decanter version. 

# Review By (Date)
- 9/20

# Criticality
- Normal

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Navigate to a page with headings and font
4. Verify the fonts are no longer than the `pro` version.
5. Verify it has the updated footer links to the Non-Discrimination and the `rel=nofollow`

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? NO
## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[-  CHEMH-247](https://stanfordits.atlassian.net/browse/CHEMH-247)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/scss/theme/_headings.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 // Headings
 h1, h2, h3, h4, h5 {
-  font-family: 'Source Sans Pro';
+  font-family: $su-font-sans;
 }
```
Comment (jdwjdwjdw) [inline src/scss/theme/_headings.scss]: Does the font-family still need to be specified here for the headings? It looks like the headings are getting this font-family inherited from the html already.

**Review by jdwjdwjdw — COMMENTED** [review]
Decanter update looks good. Just one comment / clarification.

File: src/scss/theme/_headings.scss (hunk 1 above)
Comment (jenbreese) [reply src/scss/theme/_headings.scss]: Good point.

Comment (jenbreese) [reply src/scss/theme/_headings.scss]: All done.

**Review by jdwjdwjdw — APPROVED** [review]
Thanks - looking good!

**Pull request merged**